### ISSUE TITLE
[BUG] Windows Hosts file lines too long, entries are ignored 

### ISSUE THREAD

**Issue by jduimovich** [code-ready/crc#2778]
### General information

  * OS:  Windows
  * Hypervisor:  Hyper-V  
  * Did you run `crc setup` before starting it Yes
  * Running CRC on: Desktop

## CRC version
```
CodeReady Containers version: 1.33.1+3cfaa10f
OpenShift version: 4.8.12 (bundle installed at C:\Program Files\CodeReady Containers\crc_hyperv_4.8.12.crcbundle)
```
  
## CRC status
```g:\Dev>crc status
CRC VM:          Running
OpenShift:       Running (v4.8.12)
Disk Usage:      19.41GB of 32.74GB (Inside the CRC VM)
Cache Usage:     162.9GB
Cache Directory: C:\Users\jduim\.crc\cache
```

## CRC config
```
- consent-telemetry                     : no
- cpus                                  : 8
- memory                                : 20000
```
 
### Steps to reproduce

  1.  Install CRC. It may work, it may not let you login as it cannot find oauth-openshift.apps-crc.testing depending on random --- it also seems to keep OLD namespace routes in the hosts file even if they no longer exist (this should be pruned). 
  2. Usually a clean install and delete works but as you add routes, the hosts file that is updated by CRC and as a single line which fails. Windows apparently has a limit on alias (9 per) which means as you add hosts domains fail to be found . 
  
Manually splitting the long line in hosts into multiple lines with 127.0.0.1 aliases makes it all work

CRC on windows should split the aliases into multiple lines and it will work.

**Comment by praveenkumar** [comment]
duplicate of #2710

**Comment by praveenkumar** [comment]
Closing this one, we are tracking it to original issue.

**Comment by gbraad** [comment]
darn, the message needs to be even `Duplicate of #2710` (caps)
